bridge/client/app/_models/trace.ts

```diff
@@ -158,6 +160,18 @@ class Trace {
     return this.type === EventTypes.PROBLEM_DETECTED || this.type === EventTypes.PROBLEM_OPEN;
   }
 
+  public isRemediation(): boolean {
+    return this.type.endsWith(EventTypes.REMEDIATION_TRIGGERED_SUFFIX);
+  }
+
+  public isRemediationAction(): boolean {
+    return this.type === EventTypes.ACTION_TRIGGERED;
```

Comment:
Is the type always sh.keptn.event.action.triggered only for remediation actions? It seems very generic for me - could it also be triggered with "any" action?

Reply:
for remediation, the type is always `sh.keptn.event.action.triggered`, according to KEP 37: https://github.com/keptn/enhancement-proposals/pull/37

Reply:
Ok thanks :)